tests/Feature/DoctorRegisterationTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Doctor;
+use Database\Seeders\DoctorSpecializationsSeeder;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
+use PhpParser\Comment\Doc;
+use Tests\TestCase;
+
+class DoctorRegisterationTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function setup() : void 
+    {
+        parent::setUp();
+        $this->seed(DoctorSpecializationsSeeder::class);
+    }
+
+    /** @test */
+    public function doctor_successfully_registered()
+    {
+        Storage::fake('photo');
+        Storage::fake('degree_copy');
+        $data = [
+            'name' => 'hima',
+            'email' => 'hima@gmail.com',
+            'phone' => '01225000539',
+            'specialization_id' => 1,
+            'photo' => UploadedFile::fake()->image('photo.png'),
+            'degree_copy' => UploadedFile::fake()->image('degree.png'),
+            'gender' => '1',
+            'password' => '123456789',
+        ]; 
+        $response = $this->postJson('/api/doctor/register', $data, ["Accept"=>"application/json"]);
```

Comment:
Use routes name instead of endpoints to avoid any errors if we changed endpoints path after that or exposing new APIs version
e.g. `/api/vi/register`
`/api/v2/register`

Reply:
done